common/src/main/scala/explore/cache/ProgramCache.scala

```diff
@@ -20,13 +23,19 @@ import lucuma.core.model.Observation
 import lucuma.core.model.Program
 import lucuma.core.model.Target
 import lucuma.schemas.ObservationDB
+import lucuma.schemas.ObservationDB.Enums.EditType
+import lucuma.schemas.ObservationDB.Enums.EditType.Created
+import lucuma.schemas.ObservationDB.Enums.EditType.Updated
```

Comment:
Code seems to use `EditType.Created`and `EditType.Updated`. Maybe these 2 imports can be omitted?

Reply:
It was used in a pattern match, but that's now matching on EditType.Created/Updated too